Makefile

```diff
@@ -66,6 +66,7 @@ all:
 help:
 	@echo -e "Relax-and-Recover make targets:\n\
 \n\
+  test            - Shellcheck source code\n\
   validate        - Check source code\n\
```

Comment:
Can we please merge `make validate` and `make test` so that we have only one test and not two?

Reply:
Sure 😃